user-routes.js

```diff
@@ -35,13 +49,28 @@ app.post('/users', function(req, res) {
 });
 
 app.post('/sessions/create', function(req, res) {
-  if (!req.body.username || !req.body.password) {
+
+  var username;
+  var userScheme;
+  
+  // The POST contains a username and not an email
+  if(req.body.username) {
```

Comment:
Good point, I was thinking yesterday that it would be better to extract it into a function. I'll fix it up and get back to you!